app/src/main/java/org/simple/clinic/bp/entry/confirmremovebloodpressure/ConfirmRemoveBloodPressureDialog.kt

```diff
@@ -87,7 +87,7 @@ class ConfirmRemoveBloodPressureDialog : AppCompatDialogFragment() {
     screenDestroys.onNext(ScreenDestroyed())
   }
 
-  override fun onAttach(context: Context?) {
+  override fun onAttach(context: Context) {
```

Comment:
Can you check if these are happening in this PR only?

Reply:
I checked, it seems that on my machine, the Gradle task is using `androidx.fragment:1.1.0` to resolve the classes, which has added nullability annotations. I do not know where this dependency is coming from.

Can we merge this in for the moment? It will not cause any problems in the build since they are just added nullability annotations. This should get fixed if we migrate the app to explicitly depend on `androidx` v1.1.0.